### PR TITLE
Adding Truffle Scripts

### PR DESCRIPTION
**Pull request by salazarguille**
It contains:

- Truffle scripts (loans.takeOutLoan, loans.depositCollateral, loans.withdrawCollateral, and daipool.depositDai and daipool.withdrawDai).
- Docs for scripts.
- New Ropsten addresses.
- A preamble doc to add (as a header) in contracts when they are verified in Etherscan.

### CODE REVIEW

File: docs/scripts.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+The ```./scripts``` folder contains Truffle scripts (JS files) which can be executed in any Ethereum testnet or mainnet.
+
+## Prerequisites
+
+Before executing any script, verify if the Zero Collateral contracts are deployed in the network where you want to execute it.
+
+The addresses are located at ```./config/networks/NETWORK/zerocollateral.js``` where ```NETWORK``` is the network where you want to execute the scripts.
+
+> The network names are configured in the Truffle config file. Please, see details ```./truffle-config.js``` file (section ```networks```).
+> If the contracts are not defined in the network you are looking for, please contact any team member.
```
Comment (alicevhenshaw) [inline docs/scripts.md:12]: `If the contracts are not defined in the network `
-->
`If the contracts have not been deployed on the network `

File: docs/scripts.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Executing a Script
+
+Once you verify the contracts are already deployed in the network, the next step is to execute it.
```
Comment (alicevhenshaw) [inline docs/scripts.md:16]: `in` -> `on`

File: docs/scripts.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Truffle Scripts
+
+The ```./scripts``` folder contains Truffle scripts (JS files) which can be executed in any Ethereum testnet or mainnet.
+
+## Prerequisites
+
+Before executing any script, verify if the Zero Collateral contracts are deployed in the network where you want to execute it.
```
Comment (alicevhenshaw) [inline docs/scripts.md:7]: `in` -> `on`

File: scripts/loans_takeoutLoan.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+        const hashedLoan = hashLoan({
+            interestRate,
+            collateralRatio,
+            borrower,
+            maxLoanAmount,
+            numberDays,
+            signerNonce,
+        });
+        const msg = ethUtil.bufferToHex(new Buffer(hashedLoan, 'utf8'));
```
Comment (alicevhenshaw) [inline scripts/loans_takeoutLoan.js:56]: There is a helper function to generate the signature for a loan in `hashes.js`. Is there a reason you don't use that?

Comment (salazarguille) [reply scripts/loans_takeoutLoan.js:56]: As we talked, I modified it. Now we use web3.

Comment (alicevhenshaw) [reply scripts/loans_takeoutLoan.js:56]: Will you use the function here now?

**Pull request merged**